service.py

```diff
@@ -1,18 +1,20 @@
 # Credit to https://gist.github.com/qwexvf/26215f8d5ead61ba65af013dd00c75a5
 # for parts of the login code
 import logging
-from flask import Flask, redirect, request
+from flask import Flask, redirect, request, Response
 from flask_cors import CORS
 from urllib import parse
 import requests
+from steam_api import SteamAPI
 
 
 app = Flask(__name__)
 stream_handler = logging.StreamHandler()
-stream_handler.setLevel(logging.WARNING)
+stream_handler.setLevel(logging.DEBUG)
```

Comment:
Did we want this to use en environment variable instead for future use?

Reply:
Yes, I will change this to an environment variable and add it to the README as well